kola/tests/misc/nfs.go

```diff
@@ -19,54 +19,28 @@ import (
 	"path"
 	"time"
 
-	"github.com/coreos/coreos-cloudinit/config"
-
 	"github.com/coreos/mantle/kola/cluster"
 	"github.com/coreos/mantle/kola/register"
 	"github.com/coreos/mantle/platform/conf"
 	"github.com/coreos/mantle/util"
 )
 
 var (
-	nfsserverconf = config.CloudConfig{
-		CoreOS: config.CoreOS{
-			Units: []config.Unit{
-				config.Unit{
-					Name:    "rpc-statd.service",
-					Command: "start",
-				},
-				config.Unit{
-					Name:    "rpc-mountd.service",
-					Command: "start",
-				},
-				config.Unit{
-					Name:    "nfsd.service",
-					Command: "start",
-				},
-			},
-		},
-		WriteFiles: []config.File{
-			config.File{
-				Content: "/tmp	*(ro,insecure,all_squash,no_subtree_check,fsid=0)",
-				Path: "/etc/exports",
-			},
-		},
-		Hostname: "nfs1",
-	}
-
-	mounttmpl = `[Unit]
-Description=NFS Client
-After=network-online.target
-Requires=network-online.target
-After=rpc-statd.service
-Requires=rpc-statd.service
-
-[Mount]
-What=%s:/tmp
-Where=/mnt
-Type=nfs
-Options=defaults,noexec,nfsvers=%d
-`
+	nfsserverconf = conf.ContainerLinuxConfig(`storage:
+  files:
+    - filesystem: "root"
+      path: "/etc/hostname"
+      contents:
+        inline: "nfs1"
+      mode: 0644
+    - filesystem: "root"
+      path: "/etc/exports"
+      contents:
```

Comment:
Also mode 0644 for future-proofing.

Reply:
Pushed a change to add this. Good to merge on my end.